mapie/classification.py

```diff
@@ -838,6 +838,23 @@ def _get_last_included_proba(
             axis=1
         )
 
+        zeros_scores = y_pred_proba_sorted_cumsum[:, -2, 0] == 1
+
+        if np.sum(zeros_scores) > 0:
```

Comment:
I think it would be nice to add a comment here or make this into a method to clearly explain what this does. It's a bit difficult to grasp what is going on here.

Reply:
changed